packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -90,6 +90,12 @@ export interface DatepickerProps extends UIComponentProps, Partial<ICalendarStri
 
   /** Controls the calendar's 'selectedDate'. */
   selectedDate?: Date;
+
+  /** Marks that the datepicker should only contain input and not the icon. */
+  inputOnly?: boolean;
+
+  /** Marks that the datepicker should only contain icon and not the input. */
+  iconOnly?: boolean;
```

Comment:
```suggestion
  buttonOnly?: boolean;
```
Here I'm not sure @layershifter @mnajdova , WDYT?